network/src/peer.rs

```diff
@@ -589,16 +614,28 @@ impl<N: Network, E: Environment> Peer<N, E> {
                                         Ok(expected_block_hash) => Some(expected_block_hash != block_hash),
                                         Err(_) => None,
                                     };
+
+                                    // Stop the clock on internal RTT.
+                                    #[cfg(any(feature = "test", feature = "prometheus"))]
+                                    metrics::histogram!(metrics::internal_rtt::PING, rtt_start.elapsed());
+
                                     // Send a `Pong` message to the peer.
                                     if let Err(error) = peer.send(Message::Pong(is_fork, Data::Object(ledger_reader.latest_block_locators()))).await {
                                         warn!("[Pong] {}", error);
                                     }
                                 },
                                 Message::Pong(is_fork, block_locators) => {
+                                    // Unfortunately can't be feature-flagged because of the enum
+                                    // it's passed around in.
+                                    let _rtt_start_instant: Option<Instant> = None;
+
+                                    #[cfg(any(feature = "test", feature = "prometheus"))]
+                                    let _rtt_start_instant = Some(rtt_start);
```

Comment:
nit: Can you wrap this in a scope, so it is easily comment-out-able
```rs
let _foo = {
    #[cfg(not)]
    bar
    #[cfg]
    baz
};
```

Reply:
I can add this in, though the scoping unfortunately won't make it comment-out-able because `_rtt_start_instant` is passed around in the enum (hence the variable shadowing when the feature flag is set).

Reply:
The easiest way to comment out the functionality is just by not running with `--features prometheus`.